NewTutorials/Tutorial/PlayingVideosViewController.swift

```diff
@@ -0,0 +1,96 @@
+//  Copyright © 2017 Oath. All rights reserved.
+
+import UIKit
+import OneMobileSDK
+import PlayerControls
+
+func setup(playingVideoViewController vc: PlayingVideosViewController) {
+    vc.props = .init(
+        sections: [
+            .init(rows: [
+                TextCell.ViewModel.init(name: "Single video", action: {
+                    vc.run(player: singleVideo())
+                }),
+                TextCell.ViewModel.init(name: "Array of videos", action: {
+                    vc.run(player: arrayOfVideos())
+                }),
+                TextCell.ViewModel.init(name: "Video playlist", action: {
+                    vc.run(player: videoPlaylist())
+                }),
+                TextCell.ViewModel.init(name: "Muted video", action: {
+                    vc.run(player: singleVideo(isMuted: true))
+                }),
+                TextCell.ViewModel.init(name: "Video without autoplay", action: {
+                    vc.run(player: singleVideo(isAutoplay: false))
+                })])
+        ])
+}
+
+class PlayingVideosViewController: UITableViewController {
```

Comment:
Bad name - why not just `TutorialCasesViewController`?

Reply:
`PlayingVideos` is tutorial name. Next one will be `CustomUX`